static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -533,64 +539,100 @@ require.def(
             },
 
             _getClampedTimeForPlayFrom: function (seconds) {
+                if (this._currentPlayer === this.PlayerEmps.StreamingPlayer && !this._updatingTime) {
+                    this._updateRange();
+                }
                 var clampedTime = this._getClampedTime(seconds);
                 if (clampedTime !== seconds) {
                     RuntimeContext.getDevice().getLogger().debug("playFrom " + seconds+ " clamped to " + clampedTime + " - seekable range is { start: " + this._range.start + ", end: " + this._range.end + " }");
                 }
                 return clampedTime;
             },
+            
+            _getClampOffsetFromConfig: function() {
+                var clampOffsetFromEndOfRange;
+                var config = RuntimeContext.getDevice().getConfig();
+                if (config && config.streaming && config.streaming.overrides) {
+                    clampOffsetFromEndOfRange = config.streaming.overrides.clampOffsetFromEndOfRange;
+                }
+
+                if(clampOffsetFromEndOfRange !== undefined) {
+                    return clampOffsetFromEndOfRange;
+                } else if (this._isLiveMedia()) {
+                    return this.CLAMP_OFFSET_FROM_END_OF_LIVE_RANGE;
+                } else {
+                    return this.CLAMP_OFFSET_FROM_END_OF_RANGE;
+                }
+            },
 
             _registerEventHandlers: function() {
                 var self = this;
                 this._playerPlugin.OnEvent = function(eventType, param1, param2) {
 
                     if (eventType !== self.PlayerEventCodes.CURRENT_PLAYBACK_TIME) {
```

Comment:
No, I've overwritten this method intentionally because it turned out in aging test that jumps till the end of live stream resulted later in some random bufferings when next chunk was not available yet.
This is why for live scenario PLAYFROMEND jumps to 10 seconds before end to be somewhere in the middle of last but one chunk.